hazelcast/src/main/java/com/hazelcast/config/SecurityConfig.java

```diff
@@ -121,6 +125,15 @@ public SecurityConfig setMemberCredentialsConfig(CredentialsFactoryConfig creden
         return this;
     }
 
+    public boolean getClientBlockUnmappedActions() {
```

Comment:
Can you add some javadoc here and the same description in the `hazelcast-config-3.10.xsd` and `hazelcast-spring-3.10.xsd` XSD configs?